markets/dagstore/dagstorewrapper.go

```diff
@@ -0,0 +1,174 @@
+package dagstore
```

Comment:
Rename this file to `wrapper.go`, since `dagstore` is already part of the folder and package?